Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugAuditTrail.cpp

```diff
@@ -37,7 +37,7 @@ namespace Multiplayer
         return m_filter;
     }
 
-    void MultiplayerDebugAuditTrail::SetAuditTrailFilter(const AZStd::string_view& filter)
+    void MultiplayerDebugAuditTrail::SetAuditTrailFilter(const AZStd::string filter)
     {
         m_filter = filter;
     }
```

Comment:
First this parameter shouldn't be const.

Second the filter string should be moved into m_filter.

By making the parameter const it inhibits moving of the filter parameter into m_filter
```suggestion
    void MultiplayerDebugAuditTrail::SetAuditTrailFilter(AZStd::string filter)
    {
        m_filter = AZStd::move(filter);
    }
```

Reply:
I hadn't thought of move here, I'll make that change.